### PR TITLE
#8-phone-info-creation-azadeh

### PR DESCRIPTION
**Pull request by azadehroshan**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Phone.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Phone() {
+  return (
+    <div>Phone</div>
+  )
```
Comment (Roman-Hal) [inline client/src/pages/Phone.js:6]: missing ; at the end of the line.

File: client/src/pages/Phone.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+export default Phone
```
Comment (Roman-Hal) [inline client/src/pages/Phone.js:9]: missing ; at the end of the line.

**Review by AnoshMalik — APPROVED** [review]
Wonderful work! 
Make sure the lines are ending with semi-colons please, otherwise in the future this PR will be rejected

**Review by Roman-Hal — CHANGES_REQUESTED** [review]
the file need to be corrected before merge

**Comment by youweb3** [comment]
all good

**Pull request merged**